### ISSUE TITLE
0.7.12 gives warning messages on Centos Release 6.10 

### ISSUE THREAD

**Issue by samuelxhu** [zfsonlinux/zfs#8245]
On Centos Release 6.10, kernel version 2.6.32-754.9.1.el6.x86_64
running spl/zfs 0.7.12, I encountered the following WARNING messages:

Is there a bug fix to get around these messages, or i can safely ignore...

```
WARNING: at include/linux/blkdev.h:529 blk_queue_flag_set+0x63/0x70 [zfs]() (Tainted: P           -- ------------   )
Hardware name: Super Server
Modules linked in: xt_conntrack ip6table_filter ip6_tables scst_vdisk(U) libcrc32c qla2x00tgt(U) scst(U) dlm configfs autofs4 lockd sunrpc cpufreq_ondemand acpi_cpufreq freq_table mperf ipv6 nf_conntrack_ipv4 nf_defrag_ipv4 xt_state nf_conntrack xt_multiport ipt_addrtype iptable_filter ip_tables microcode iTCO_wdt iTCO_vendor_support sb_edac edac_core i2c_i801 lpc_ich mfd_core joydev igb dca i2c_algo_bit i2c_core ptp pps_core shpchp zfs(P)(U) zcommon(P)(U) znvpair(P)(U) icp(P)(U) spl(U) zlib_deflate zavl(P)(U) zunicode(P)(U) cnsldev(U) ses enclosure ipmi_devintf sg power_meter acpi_ipmi ipmi_si ipmi_msghandler ext4 jbd2 mbcache raid1 sd_mod crc_t10dif xhci_hcd ahci mpt3sas scsi_transport_sas raid_class qla2xxx_scst(U) scsi_transport_fc scsi_tgt wmi dm_mirror dm_region_hash dm_log dm_mod [last unloaded: scsi_wait_scan]
Pid: 5084, comm: z_zvol Tainted: P           -- ------------    2.6.32-754.9.1.el6.x86_64 #1
Call Trace:
 [<ffffffff810805c1>] ? warn_slowpath_common+0x91/0xe0
 [<ffffffff8108062a>] ? warn_slowpath_null+0x1a/0x20
 [<ffffffffa0480ae3>] ? blk_queue_flag_set+0x63/0x70 [zfs]
 [<ffffffffa0480f25>] ? zvol_create_minor_impl+0x435/0x7f0 [zfs]
 [<ffffffffa02f7fbc>] ? taskq_wait_outstanding+0xbc/0xd0 [spl]
 [<ffffffff810aaed0>] ? autoremove_wake_function+0x0/0x40
 [<ffffffffa04817de>] ? zvol_task_cb+0x27e/0x560 [zfs]
 [<ffffffffa02f8a64>] ? taskq_thread+0x304/0x620 [spl]
 [<ffffffff81070890>] ? default_wake_function+0x0/0x20
 [<ffffffffa02f8760>] ? taskq_thread+0x0/0x620 [spl]
 [<ffffffff810aaa30>] ? kthread+0xa0/0xc0
 [<ffffffff815657d0>] ? child_rip+0x20/0x30
 [<ffffffff810aa990>] ? kthread+0x0/0xc0
 [<ffffffff815657b0>] ? child_rip+0x0/0x30
---[ end trace 918c153eaaa8a710 ]---
------------[ cut here ]------------
WARNING: at include/linux/blkdev.h:546 zvol_create_minor_impl+0x76a/0x7f0 [zfs]() (Tainted: P        W  -- ------------   )
Hardware name: Super Server
Modules linked in: xt_conntrack ip6table_filter ip6_tables scst_vdisk(U) libcrc32c qla2x00tgt(U) scst(U) dlm configfs autofs4 lockd sunrpc cpufreq_ondemand acpi_cpufreq freq_table mperf ipv6 nf_conntrack_ipv4 nf_defrag_ipv4 xt_state nf_conntrack xt_multiport ipt_addrtype iptable_filter ip_tables microcode iTCO_wdt iTCO_vendor_support sb_edac edac_core i2c_i801 lpc_ich mfd_core joydev igb dca i2c_algo_bit i2c_core ptp pps_core shpchp zfs(P)(U) zcommon(P)(U) znvpair(P)(U) icp(P)(U) spl(U) zlib_deflate zavl(P)(U) zunicode(P)(U) cnsldev(U) ses enclosure ipmi_devintf sg power_meter acpi_ipmi ipmi_si ipmi_msghandler ext4 jbd2 mbcache raid1 sd_mod crc_t10dif xhci_hcd ahci mpt3sas scsi_transport_sas raid_class qla2xxx_scst(U) scsi_transport_fc scsi_tgt wmi dm_mirror dm_region_hash dm_log dm_mod [last unloaded: scsi_wait_scan]
Pid: 5084, comm: z_zvol Tainted: P        W  -- ------------    2.6.32-754.9.1.el6.x86_64 #1
Call Trace:
 [<ffffffff810805c1>] ? warn_slowpath_common+0x91/0xe0
 [<ffffffff8108062a>] ? warn_slowpath_null+0x1a/0x20
 [<ffffffffa048125a>] ? zvol_create_minor_impl+0x76a/0x7f0 [zfs]
 [<ffffffff810aaed0>] ? autoremove_wake_function+0x0/0x40
 [<ffffffffa04817de>] ? zvol_task_cb+0x27e/0x560 [zfs]
 [<ffffffffa02f8a64>] ? taskq_thread+0x304/0x620 [spl]
 [<ffffffff81070890>] ? default_wake_function+0x0/0x20
 [<ffffffffa02f8760>] ? taskq_thread+0x0/0x620 [spl]
 [<ffffffff810aaa30>] ? kthread+0xa0/0xc0
 [<ffffffff815657d0>] ? child_rip+0x20/0x30
 [<ffffffff810aa990>] ? kthread+0x0/0xc0
 [<ffffffff815657b0>] ? child_rip+0x0/0x30
---[ end trace 918c153eaaa8a711 ]---
 zd0: unknown partition table
 zd16: unknown partition table
 zd32: unknown partition table
```

**Comment by kpande** [comment]
you ignored the issue template and submitted a support request instead of a bug which is meant for [zfs-discuss](http://list.zfsonlinux.org).

**Comment by behlendorf** [comment]
You can safely ignore, but we should see if we can safely address this warning for this old kernel.

**Comment by Ornias1993** [comment]
@behlendorf 
Considering 6.0 is dropped for future versions, what is the current policy for support of older ZoL versions on CentOS 6.0?

**Comment by behlendorf** [comment]
New point releases for the 0.8.x series will continue to support CentOS 6, and we'll backport existing fixes as appropriate.

But this is a good example of an issue it makes sense to close out, the warning was resolved by 10f88c5cd523eec8431941abf00017fc8fb67fd3 which is included in the 0.8.x releases.
